### PR TITLE
Add support for --ignorefile option (.trivyignore)

### PR DESCRIPTION
**Pull request by achton**
This PR adds support for supplying multiple ignorefiles to the action.

It also adds a test for the functionality, and adjusts a few minor test-related things. 👼🏻 

See #103 for the feature request.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -114,6 +117,21 @@ if [ $skipDirs ];then
     SARIF_ARGS="$SARIF_ARGS --skip-dirs $i"
   done
 fi
+if [ $trivyIgnores ];then
+  for f in $(echo $trivyIgnores | tr "," "\n")
+  do
+    touch ./trivyignores
```
Comment (simar7) [inline entrypoint.sh:123]: Just curious why do we need to `touch` the file?

**Comment by simar7** [comment]
thanks for the PR! looks great, just seems that the tests are failing at the moment.

File: entrypoint.sh (hunk 1 above)
Comment (achton) [reply entrypoint.sh]: @simar7 You are right, we don't. I've removed that line :+1:

**Review by simar7 — APPROVED** [review]
thanks lgtm.

**Pull request merged**